python/src/robyn/modeling/entities/clustering_results.py

```diff
@@ -1,64 +1,87 @@
 # pyre-strict
-
 from dataclasses import dataclass
+from typing import Dict, List, Optional
+
+import matplotlib.pyplot as plt
+
 import pandas as pd
-import numpy as np
-from typing import List
 
 
 @dataclass
-class KMeansResults:
+class PlotResults:
```

Comment:
if these are all generated during clustering, perhaps we can consider renaming this to ClusteringPlotResults?